packages/block-library/src/post-terms/edit.js

```diff
@@ -83,8 +83,7 @@ export default function PostTermsEdit( {
 			</BlockControls>
 			<InspectorControls group="advanced">
 				<TextControl
-					// TODO: Switch to `true` (40px size) if possible
-					__next40pxDefaultSize={ false }
+					__next40pxDefaultSize
```

Comment:
Can be seen in the Categories block.

| Before | After |
|--------|-------|
|<img src="https://github.com/user-attachments/assets/52abb3ab-546c-4968-9478-b5e12df49cd1" alt="Category block inspector, before" width="262">|<img src="https://github.com/user-attachments/assets/ed4d703e-5bf4-4693-b6a5-be3cdcd91318" alt="Category block inspector, after" width="262">|